.eslintrc.js

```diff
@@ -68,5 +69,12 @@ module.exports = {
         'react-hooks/rules-of-hooks': 'off',
       },
     },
+    {
+      files: ['**/*.stories.*'],
+      rules: {
+        'react/display-name': 'off',
```

Comment:
Is this causing problems somehow?

Reply:
Wouldn't know how this might causes issues really.